dataverse-webapp/src/main/webapp/viewMetadata.xhtml

```diff
@@ -6,24 +8,29 @@
                 xmlns:p="http://primefaces.org/ui"
                 xmlns:o="http://omnifaces.org/ui"
                 xmlns:jsf="http://xmlns.jcp.org/jsf">
-    
+
     <!--@elvariable id="metadataBlocks" type="java.util.List<edu.harvard.iq.dataverse.persistence.dataset.MetadataBlock>"-->
     <!-- View Mode -->
     <div class="panel-group form-horizontal">
         <ui:repeat value="#{metadataBlocks}"
                    var="metadataBlockVal" varStatus="block">
             <div class="panel panel-default">
-                <a data-toggle="collapse" href="#panelCollapse#{block.index}" class="panel-heading text-info" tabindex="0"
+                <a id="clps#{block.index}" data-toggle="collapse" href="#panelCollapse#{block.index}" class="collapseSrc panel-heading text-info" tabindex="0"
                         aria-expanded="#{block.first}">
-                    <h3>#{metadataBlockVal.key.localeDisplayName} &#160;<span class="glyphicon #{block.first?'glyphicon-chevron-up':'glyphicon-chevron-down'}" aria-hidden="true"/></h3>
+                    <h3>#{metadataBlockVal.key.localeDisplayName} &#160;<span class="collapseSrc glyphicon #{block.first?'glyphicon-chevron-up':'glyphicon-chevron-down'}" aria-hidden="true"/></h3>
                 </a>
                 <div id="panelCollapse#{block.index}" class="collapse #{block.first?'in':''}">
+                    <script>
+                        $('#panelCollapse#{block.index}').on('shown.bs.collapse', function() {
+                            Drodb.Geo.MetadataView.initializeAll('map_#{metadataBlockVal.key.name}');
```

Comment:
From what I can see `initializeAll` should be executed after `prepare`. What will happen `GEOBOX` will be in citation (in metadata block that is expanded by default)? I don't think that event will be executed in such case